controller/api_test.go

```diff
@@ -282,6 +282,15 @@ func TestMain(m *testing.M) {
 	resp, err := http.Get("http://localhost:8888/")
 	panicIf(err)
 	assert(resp.StatusCode == 200, "http get status code on root")
+
```

Comment:
This is only testing the homehandler; could you also test one of the other endpoints?